geomesa-feature/pom.xml

```diff
@@ -36,9 +36,9 @@
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.esotericsoftware</groupId>
-            <artifactId>kryo-shaded</artifactId>
-            <version>3.0.0</version>
+            <groupId>com.esotericsoftware.kryo</groupId>
+            <artifactId>kryo</artifactId>
+            <version>2.21</version>
```

Comment:
Good question.  I've rebuilt (with tests), and not seen any complaints.  As a check, I ran the following:

```
11:32 $ mvn dependency:tree | grep -i kryo
[INFO] +- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:provided
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:provided
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
```

Looks reasonable to me, but if I'm overlooking something, please just holler.